Entities/Items/Crate/Crate.as

```diff
@@ -25,14 +25,15 @@ void onInit(CBlob@ this)
 	this.addCommandID("stop unpack");
 	this.addCommandID("boobytrap");
 
+	string packed = this.get_string("packed");
+
 	this.set_u32("boobytrap_cooldown_time", 0);
-	this.set_s32("gold building amount", 0);
+	this.set_s32("gold building amount", packed == "warboat" ? 50 : 0);
```

Comment:
I'd rather move from a ternary to an if branch because adding cases would be super uglier with the ternary.
```suggestion
	int goldAmount = 0;
	if (packed == "warboat")
	{
		goldAmount = 50;
	}
	this.set_s32("gold building amount", goldAmount);
```